src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/Main.cs

```diff
@@ -72,8 +77,8 @@ public List<Result> Query(Query query)
                             foreach (var searchResult in searchResultsList)
                             {
                                 var path = searchResult.Path;
-                                var toolTipTitle = string.Format("{0} : {1}", _context.API.GetTranslation("Microsoft_plugin_indexer_name"), searchResult.Title);
-                                var toolTipText = string.Format("{0} : {1}", _context.API.GetTranslation("Microsoft_plugin_indexer_path"), path);
+                                var toolTipTitle = string.Format(CultureInfo.InvariantCulture, "{0} : {1}", _context.API.GetTranslation("Microsoft_plugin_indexer_name"), searchResult.Title);

```

Comment:
Shouldn't we use CurrentCulture here since it is an actual translation? Same below